wled00/NpbWrapper.h

```diff
@@ -258,7 +267,7 @@ class NeoPixelWrapper
           #endif
         }
         analogWriteRange(255);  //same range as one RGB channel
-        analogWriteFreq(880);   //PWM frequency proven as good for LEDs
+        analogWriteFreq(PWM_FREQUENCY);   //PWM frequency proven as good for LEDs
```

Comment:
just a drive-by comment as I saw this mentioned on discord, but the comment on this line now seems out of place, and perhaps a comment should be at the definition?  (I'm not a WLED dev, just browsing.)